Cleanup documentation (#2)
ScopeLift/flexible-voting-clients: README.md

# Flexible Voting Clients 

> [Flexible Voting](https://github.com/ScopeLift/flexible-voting) is an extension to the widely used OpenZeppelin DAO Governor that enables novel voting patterns for delegates. Clients are contracts that leverage Flexible Voting to allow they these novel voting patterns, such as:

Reviewer: ```suggestion
[Flexible Voting](https://github.com/ScopeLift/flexible-voting) is an extension to the widely used [OpenZeppelin Governor](TODO add link to docs here) that enables novel voting patterns for delegates. Clients are contracts that leverage Flexible Voting to allow they these novel voting patterns, such as:
```

Author: https://github.com/ScopeLift/flexible-voting-clients/pull/2/commits/373634678e35211a826b30c72694a589aef0fe8a

  - Voting with tokens while earning yield in DeFi
  - Voting with tokens bridged to L2
  - Shielded voting (i.e. secret/private voting)
  - Cheaper subsidized signature based voting

>   - Better voting options with tokens held by custodians

Reviewer: What does this last bullet mean?

Author: https://github.com/ScopeLift/flexible-voting-clients/pull/2/commits/373634678e35211a826b30c72694a589aef0fe8a

> Flexible Voting Clients contains existing clients and the tools to build new clients.

Reviewer: I don't understand what this sentence means

Author: How about "The Flexible Voting Clients repo contains existing clients and the necessary tools to build new clients."? I am basically trying to say it contains client implementations and helper contracts for people to build their own clients.

Reviewer: That sgtm, I didn't realize the first part of the sentence was referring to the repo, makes sense this way. Feel free to resolve once the change is made

Author: https://github.com/ScopeLift/flexible-voting-clients/commit/a1360f71c59b43876407ff588e5e8676eb04f7fc

#### Getting started

Clone the repo

```bash
git clone git@github.com:ScopeLift/flexible-voting-clients.git
cd flexible-voting-clients
```

Copy the `.env.template` file and populate it with values

```bash

> cp sample .env

Reviewer: ```suggestion
cp .env.template .env
```

Author: https://github.com/ScopeLift/flexible-voting-clients/pull/2/commits/373634678e35211a826b30c72694a589aef0fe8a

# Open the .env file and add your values
```

```bash
forge install
forge build
forge test
```

#### Repo Contents

| Name               | Description                                                                                                                                                                                    | Contract                                           |
|--------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------|

> | Aave Atoken Client | An extension of Aave V3's AToken contract which makes it possible for AToken  holders to still vote on governance proposals                                                                    | [ATokenFlexVoting](./src/ATokenFlexVoting.sol)     |

Reviewer: ```suggestion
| Aave Atoken Client | An extension of Aave V3's AToken contract which makes it possible for AToken holders to still vote on governance proposals                                                                    | [ATokenFlexVoting](./src/ATokenFlexVoting.sol)     |
```

Author: https://github.com/ScopeLift/flexible-voting-clients/pull/2/commits/373634678e35211a826b30c72694a589aef0fe8a

| Comet Client       | This is an extension of Compound V3's Comet contract which makes it possible for Comet token holders to still vote on governance proposals.                                                    | [CometFlexVotingClient](./src/CometFlexVoting.sol) |

> | Flex Voting Client |This is an abstract contract to make it easy to build clients for governance systems that are "flexible voting" governors. | [FlexVotingClient](./src/FlexVotingClient.sol) |

Reviewer: ```suggestion
| Flex Voting Client |This is an abstract contract to make it easy to build clients for governance systems that are "flexible voting" governors. The above clients inherit from this contract. | [FlexVotingClient](./src/FlexVotingClient.sol) |
```

Author: https://github.com/ScopeLift/flexible-voting-clients/pull/2/commits/373634678e35211a826b30c72694a589aef0fe8a
